CondTools/DT/src/DTPerformanceHandler.cc

```diff
@@ -128,8 +128,10 @@ void DTPerformanceHandler::getNewObjects() {
   cond::Time_t snc = runNumber;
   if (runNumber > last)
```

Comment:
@cms-sw/db-l2 can this check be moved to the start of the function?

Reply:
to the extend that it may matter, DT DPG is fine with the proposal